matter_server/server/helpers/paa_certificates.py

```diff
@@ -22,10 +23,57 @@
 PRODUCTION_URL = "https://on.dcl.csa-iot.org"
 TEST_URL = "https://on.test-net.dcl.csa-iot.org"
 GIT_URL = "https://github.com/project-chip/connectedhomeip/raw/master/credentials/development/paa-root-certs"  # pylint: disable=line-too-long
```

Comment:
We can use the git repo details below to construct this url.

Reply:
Ack. Testing changes now